tailblocks/src/components/About/AboutUs.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import About from '../../pages/about';
```

Comment:
Looks like this path doesn't exist anymore. Check the case in each import. It causes issues on Mac and Linux.

Reply:
Removed all unnecessary imports.